frontend/apps/app/features/projects/utils/getOriginalDocumentContent.ts

```diff
@@ -1,5 +1,5 @@
 import { getFileContent } from '@liam-hq/github'
-import { getProjectRepository } from '../../../app/(app)/app/projects/[projectId]/ref/[branchOrCommit]/docs/[...docFilePath]/getProjectRepository'
+import { getProjectRepository } from '../../../app/(app)/app/(with-project-and-branch)/projects/[projectId]/ref/[branchOrCommit]/docs/[...docFilePath]/getProjectRepository'
```

Comment:
Not directly related to this PR, but the dependency direction here is a bit unusual in this project?